model/Licensing/Properties/standardLicenseHeader.md

```diff
@@ -4,11 +4,15 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-TODO
+A standardLicenseHeader provides a License author's preferred text to
+indicate that a file is covered by the License.
 
 ## Description
 
-TODO
+A standardLicenseHeader contains the plain text of the License author's
+preferred language to be used, typically in a source code file's header
```

Comment:
```suggestion
A standardLicenseHeader contains the plain text of the License author's
preferred wording to be used, typically in a source code file's header
```
"language" is a kind of overloaded word in the software world ;)

Reply:
Fair point  :)